frontend/src/employee-frontend/components/child-information/placements/NewServiceNeeds.tsx

```diff
@@ -36,28 +83,237 @@ function NewServiceNeeds({ serviceNeeds }: Props) {
+    form.optionId &&
+    !form.endDate.isBefore(form.startDate)
+
+  const onSubmit = () => {
+    if (form.startDate && form.endDate && form.optionId) {
+      setSubmitting(true)
+
+      if (editingId) {
+        void updateNewServiceNeed(editingId, {
+          startDate: form.startDate,
+          endDate: form.endDate,
+          optionId: form.optionId,
+          shiftCare: form.shiftCare
+        })
+          .then((res) => {
+            if (res.isSuccess) {
+              onSuccess()
+            } else {
+              setErrorMessage({
+                type: 'error',
+                title: i18n.common.error.unknown,
+                text: i18n.common.error.saveFailed,
+                resolveLabel: i18n.common.ok
+              })
+            }
+          })
+          .finally(() => setSubmitting(false))
+      } else {
+        void createNewServiceNeed({
+          placementId: placement.id,
+          startDate: form.startDate,
+          endDate: form.endDate,
+          optionId: form.optionId,
+          shiftCare: form.shiftCare
+        })
+          .then((res) => {
+            if (res.isSuccess) {
+              onSuccess()
+            } else {
+              setErrorMessage({
+                type: 'error',
+                title: i18n.common.error.unknown,
+                text: i18n.common.error.saveFailed,
+                resolveLabel: i18n.common.ok
+              })
+            }
+          })
+          .finally(() => setSubmitting(false))
+      }
+    }
+  }
+
+  return (
+    <Tr>
+      <Td>
+        <FixedSpaceRow spacing="xs">
+          <DatePickerDeprecated
+            date={form.startDate}
+            onChange={(date) => setForm({ ...form, startDate: date })}
+            minDate={placement.startDate}
+            maxDate={placement.endDate}
+          />
+          <span>-</span>
+          <DatePickerDeprecated
+            date={form.endDate}
+            onChange={(date) => setForm({ ...form, endDate: date })}
+            minDate={placement.startDate}
+            maxDate={placement.endDate}
+          />
+        </FixedSpaceRow>
+      </Td>
+      <Td>
+        <SimpleSelect
+          options={options}
+          value={form.optionId}
+          onChange={(e) => setForm({ ...form, optionId: e.target.value })}
+          placeholder={t.optionPlaceholder}
+        />
+      </Td>
+      <Td>
+        <Checkbox
+          label={t.shiftCare}
+          hiddenLabel
+          checked={form.shiftCare}
+          onChange={(checked) => setForm({ ...form, shiftCare: checked })}
+        />
+      </Td>
+      <Td>
+        <FixedSpaceRow justifyContent="flex-end">
+          <InlineButton
+            onClick={onCancel}
+            text={i18n.common.cancel}
+            disabled={submitting}
+          />
+          <InlineButton
+            onClick={onSubmit}
+            text={i18n.common.save}
+            disabled={submitting || !formIsValid}
+          />
+        </FixedSpaceRow>
```

Comment:
I would add a slightly larger space between the buttons, it didn't feel good when I used the editor. The ui spec also has a `24px` margin between them.